dashboard/app/models/circuit_playground_discount_application.rb

```diff
@@ -28,15 +28,14 @@ class CircuitPlaygroundDiscountApplication < ApplicationRecord
 
   enum unit_6_intention: {
     no: 1,
-    yes1718: 2,
-    yes1819: 3,
-    yesAfter: 4,
     unsure: 5,
-    yes1920: 6,
+    yesSpring2020: 7,
+    yesFall2020: 8,
+    yesSpring2021: 9
```

Comment:
I'd like us to leave the old enum values in place here, since we've still got the old application records in our database and we store this value as an integer.  Note this is the _only_ place I think we need to keep them - I'm fine with the matching change you made in discountLogic.js to include only the values that might be submitted this year.

Reply:
Thought this might be the wrong decision :) Fixed!